plugin/build.gradle.kts

```diff
@@ -53,6 +53,15 @@ pluginBundle {
     }
 }
 
+configure<PublishingExtension> {
+    repositories {
+        maven {
+            name = "pluginTest"
+            url = uri("file://${rootProject.buildDir}/localMaven")
+        }
+    }
```

Comment:
Just leave a comment above this block to explain that we're publishing inside the `pluginTest` maven repo for test purposes. Otherwise could look harder to understand for external contributors.